deno.json

```diff
@@ -0,0 +1,6 @@
+{
+  "tasks": {
+    "format": "deno run --unstable --allow-write --allow-read --allow-run ./tools/format.js",
+    "lint": "deno run --unstable --allow-write --allow-read --allow-run ./tools/lint.js"
+  }
+}
```

Comment:
We can't do this because it will break many tests since this config file will now be auto-discovered by them.

Reply:
I'm not sure how 2 tasks would break testing... Are you confusing the `tasks` field with the `format` and `lint` fields?

If the configuration file does more harm than good, I'll close this PR.

Reply:
Deno has different behaviour whether a deno.json file exists or not. For example, a lockfile gets automatically generated if a deno.json exists. Although it would be nice to add this (we also tried to do it in the past), I don't think it's worth it because it affects the how the tests get run if run from the testdata directory.

Reply:
Ok. I see. If that's the case, I'll close this case. But out of curiosity, how does the configuration affect tests?

Reply:
The integration tests are in ./cli/tests/testdata and right now many of them require the cwd to be in that directory, so when a test gets run with that as the CWD, then it will resolve the config file at ./deno.json. Overall it's just easier not to have a deno.json in the repo even though it would be nice.

Reply:
Ah, I see. Thanks for the explanation! Makes sense.